src/main.dev.ts

```diff
@@ -96,6 +99,36 @@ if (isWindows) {
       }
     );
   }
+
+  process.env.PLATFORMIO_INSTALLER_TMPDIR = userDataDirectory;
```

Comment:
I think we should pass env parameters to platformio subprocess through `localApiServerEnv` instead of assigning it directly to `process.env` of parent process.